aptos-move/framework/aptos-framework/sources/account.spec.move

```diff
@@ -301,36 +322,34 @@ spec aptos_framework::account {
 
         // Verify all properties in update_auth_key_and_originating_address_table
         let originating_addr = addr;
-        let new_auth_key_vector = spec_assert_valid_rotation_proof_signature_and_get_auth_key(to_scheme, to_public_key_bytes, cap_update_table, challenge);
+        let new_auth_key_vector = spec_assert_valid_rotation_proof_signature_and_get_auth_key(
+            to_scheme, to_public_key_bytes, cap_update_table, challenge);
 
         let address_map = global<OriginatingAddress>(@aptos_framework).address_map;
         let new_auth_key = from_bcs::deserialize<address>(new_auth_key_vector);
 
         aborts_if !exists<OriginatingAddress>(@aptos_framework);
         aborts_if !from_bcs::deserializable<address>(account_resource.authentication_key);
-        aborts_if table::spec_contains(address_map, curr_auth_key) &&
-            table::spec_get(address_map, curr_auth_key) != originating_addr;
+        aborts_if table::spec_contains(address_map, curr_auth_key) && table::spec_get(
```

Comment:
The original code looks better.

Reply:
In general, breaking to newline after an operator is preferable to breaking to newline in the middle of a call (like in the original).